clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -448,7 +906,7 @@ Expected<StringRef> clang(ArrayRef<StringRef> InputFiles, const ArgList &Args) {
 } // namespace generic
 
 Expected<StringRef> linkDevice(ArrayRef<StringRef> InputFiles,
-                               const ArgList &Args) {
+                               const ArgList &Args, bool IsSYCLKind = false) {
```

Comment:
Nit: Can we use an enum and no default argument?

Reply:
This seems more clear to me. Thanks

Reply:
Why do we need the flag in the first place?

Before the PR, `clang-linker-wrapper` would emit error if targeted for `spir64`, now it either does some handling if in SYCL mode, or it emits an empty module. Why can't we just make `spir64` always mean SYCL?

Reply:
I will need to think some more about this. But I think that it should not hold up the PR. 

Thanks